src/library/trackset/crate/cratetablemodel.cpp

```diff
@@ -178,3 +181,22 @@ void CrateTableModel::removeTracks(const QModelIndexList& indices) {
 
     select();
 }
+
+QString CrateTableModel::modelKey(bool noSearch) const {
+    if (this->m_selectedCrate.isValid()) {
+        if (noSearch) {
+            return QStringLiteral("crate/") +
```

Comment:
Both '/' and ':' are used as separators. I don't see a consistent pattern?

Reply:
I'l use `:` everywhere